src/utils/queryParamsStringify.ts

```diff
@@ -0,0 +1,50 @@
+import type { HTTPQuery } from '../pinecone-generated-ts-fetch';
+
+export function queryParamsStringify(
```

Comment:
I copied most of the code in this file from the [generated client code](https://github.com/pinecone-io/pinecone-ts-client/blob/main/src/pinecone-generated-ts-fetch/runtime.ts#L301)

Reply:
You may want to leave a comment here explaining that + linking to the source